client/js/components/prevent-overlapping.js

```diff
@@ -0,0 +1,42 @@
+import {nodeList, featureTest} from '../util';
+
+const preventOverlapping = (els) => {
+  if (!featureTest('position', 'sticky')) {
+    return;
+  }
+  const elements = nodeList(els).reverse();
+  const hideOverlapping = () => {
+    function getElementMeta() {
```

Comment:
Could we make this a function that takes arguments instead of defining it inline?